next.config.js

```diff
@@ -33,6 +39,24 @@ module.exports = (phase, { defaultConfig }) => {
 
       return config
     },
+    async headers() {
+      return [
```

Comment:
can we move this config to a `headersConfig` variable at the top?

Reply:
Sure. I'll do it for the end of the day (UTC+2)